<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceWidget</class>
 <widget class="QWidget" name="ReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>122</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
     <property name="buddy">
      <cstring>comboFind</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QPushButton" name="findButton">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QRadioButton" name="radioForward">
     <property name="text">
      <string>Forward</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="texmaker.qrc">
       <normaloff>:/images/fileclose16.png</normaloff>:/images/fileclose16.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="8" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>26</width>
       <height>78</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
     <property name="buddy">
      <cstring>comboReplace</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboReplace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QPushButton" name="replaceallButton">
     <property name="text">
      <string>Replace All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkWords">
     <property name="text">
      <string>Whole words only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QCheckBox" name="checkCase">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QCheckBox" name="checkBegin">
     <property name="text">
      <string>Start at Beginning</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="moreButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="8">
    <widget class="QFrame" name="extension">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkRegExp">
        <property name="toolTip">
         <string notr="true">Captured text for the &quot;Replace&quot; field : $1 $2..</string>
        </property>
        <property name="text">
         <string>Regular Expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkSelection">
        <property name="toolTip">
         <string>Text must be selected before checking this option.</string>
        </property>
        <property name="text">
         <string>In selection only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QRadioButton" name="radioBackward">
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboFind</tabstop>
  <tabstop>comboReplace</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>radioForward</tabstop>
  <tabstop>replaceallButton</tabstop>
  <tabstop>checkWords</tabstop>
  <tabstop>checkCase</tabstop>
  <tabstop>checkBegin</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>moreButton</tabstop>
  <tabstop>checkRegExp</tabstop>
 </tabstops>
 <resources>
  <include location="texmaker.qrc"/>
 </resources>
 <connections/>
</ui>
